ui/app/AppLayouts/Chat/ChatColumn/MessageComponents/EmojiReactions.qml

```diff
@@ -77,7 +77,6 @@ Item {
                 StatusToolTip {
                     visible: mouseArea.containsMouse
                     text: showReactionAuthors(modelData.fromAccounts, modelData.emojiId)
-                    width: 400
```

Comment:
This one needs a max-width of 400, otherwise, all the emoji reactions are gonna appear on one line and it's going to be super long

Reply:
There is a problem to use the `width` attribute regarding my test. If the `StatusToolTip` has `width`, its `implicitWidth` attribute would be ignored and the final width of the `StatusToolTip` would be fixed.
If you have a good solution, please give me.

Reply:
If we keep `width` attribute, Emoji reaction tooltip has wrong left/right padding again.

Reply:
Maybe tou can just do a `Math.min` on the implicitWidth?
Something like:
```qml
implicitWidth: Math.min(maxWidth, textContent.implicitWidth + Style.current.bigPadding)
```

Reply:
What is the `maxWidth`? Is it came from `width`?

Reply:
No, sorry, you're a bit out of the loop. It,s from the design: https://www.figma.com/file/Mr3rqxxgKJ2zMQ06UAKiWL/%F0%9F%92%AC-Chat%E2%8E%9CDesktop?node-id=4005%3A38331

Basically, we can that tooltip to be maximum of 400px of width. When there is too many reactions, they just wrap on multiple lines (it does that automatically).

So you can add a new property to StatusTooltip called `maxWidth` that can be set for the emoji reactions tooltip

Reply:
Ok, I understand